config/jobs/kubernetes/sig-release/release-branch-jobs/dra-1.32.yaml

```diff
@@ -0,0 +1,234 @@
+periodics:
```

Comment:
Can you point me out to the documentation about forking jobs to release branch?

Reply:
https://github.com/kubernetes/test-infra/blob/206b4bd14d66c078a4dc476b91830dab02b5aa2f/releng/config-forker/README.md#supported-annotations